gapis/resolve/dependencygraph2/graph_visualization/graph_visualization.go

```diff
@@ -0,0 +1,21 @@
+package graph_visualization
+
+import  (
+	 "context"
+	 "github.com/google/gapid/gapis/capture"
+	 "github.com/google/gapid/gapis/resolve/dependencygraph2"
+	 "github.com/google/gapid/core/math/interval"
+	 "github.com/google/gapid/gapis/api"
+	 "github.com/google/gapid/core/log"
+ )
+
+
+func GetGraphVisualizationFileFromCapture(ctx context.Context , p *capture.Capture) (string , error) {
+
+	log.I(ctx , "Working on GetGraphDotFileFromCapture")
```

Comment:
It would be better to use the status package here. It will let us profile this. We should try to be as quiet as possible in the normal case.

Reply:
I added the status package on the server function that calls directly to this function GetGraphVisualizationFileFromCapture, since It also allows us to do the profiling.